ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -3007,11 +3007,13 @@ tests:
     cluster_profile: vsphere-elastic
     env:
       FILTERS_ADDITIONAL: ""
+      INDEX_IMAGE: quay.io/redhat-user-workloads/ocp-isc-tenant/file-integrity-operator-fbc-4-15@sha256:aa51bfd2a19330b39cd770e418d36c107daaa6aca01d3b5eea671c2d41d0bb47
```

Comment:
Should be able to update this to:

[quay.io/redhat-user-workloads/ocp-isc-tenant/file-integrity-operator-fbc-4-15@sha256:49e5b1abe12d46b33e365b2f90c8d285894667167e1b35f4b97c225e95906c4b](http://quay.io/redhat-user-workloads/ocp-isc-tenant/file-integrity-operator-fbc-4-15@sha256:49e5b1abe12d46b33e365b2f90c8d285894667167e1b35f4b97c225e95906c4b)